image.go

```diff
@@ -20,6 +20,10 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
+	// RootFS returns the image digests
+	RootFS(ctx context.Context) ([]digest.Digest, error)
+	// Size returns the image size
```

Comment:
Also, the image digests for what? These are very specific and should be called out.